README.md

```diff
@@ -9,9 +9,30 @@ permissions that are set on the workspace level, Secret scopes, Notebooks, Direc
 
 See [contributing instructions](CONTRIBUTING.md) to help improve this project.
 
+## Introduction
+UCX will guide you, the Databricks customer through the process of upgrading your account, groups, workspaces, jobs etc to Unity Catalog. 
+
+The upgrade process starts with installing code to run the upgrade assessment as well as modules for upgrading various components of the system. Component upgrades typically involve evolving configuration and metadata contained within your Databricks deployment. There are parts of the upgrade process that will recommend individual code changes for jobs. The upgrade user will be directed to trigger these operations one by one. 
```

Comment:
Can you maybe highlight that normal process is as simple as triggering the jobs created by this app?